pkg/utils/cluster.go

```diff
@@ -15,39 +15,119 @@ import (
 
 const (
 	// default Infrastructure resource name for Openshift
-	infraResourceName = "cluster"
+	infraResourceName  = "cluster"
+	workerNodeLabelKey = "node-role.kubernetes.io/worker"
+	masterNodeLabelKey = "node-role.kubernetes.io/master"
```

Comment:
what about `node-role.kubernetes.io/control-plane` ?
in k8s 1.24 kubeadm now replaced master with control-plane (master label deprecated)

Reply:
added support for both.